packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -259,16 +264,24 @@ class VideoViewport extends Viewport implements IVideoViewport {
   }
 
   public play() {
-    if (!this.isPlaying) {
-      this.videoElement.play();
-      this.isPlaying = true;
-      this.renderWhilstPlaying();
+    try {
+      if (!this.isPlaying) {
+        this.videoElement.play();
+        this.isPlaying = true;
+        this.renderWhilstPlaying();
+      }
+    } catch (e) {
+      // No-op, this happens sometimes on startup
```

Comment:
does this "this happens sometimes on startup" mean sometimes the auto-play won't work?